### ISSUE TITLE
Device not being discovered. 

### ISSUE THREAD

**Issue by 6d6176** [attify/Attify-Zigbee-Framework#4]
feels like im missing a step but, 

kb@ubuntu:~/Attify-Zigbee-Framework$ lsusb
Bus 001 Device 001: ID 1d6b:0002 Linux Foundation 2.0 root hub
Bus 002 Device 006: ID 0451:16ae Texas Instruments, Inc. 
Bus 002 Device 005: ID 03eb:210a Atmel Corp. 
Bus 002 Device 004: ID 0e0f:0008 VMware, Inc. 
Bus 002 Device 003: ID 0e0f:0002 VMware, Inc. Virtual USB Hub
Bus 002 Device 002: ID 0e0f:0003 VMware, Inc. Virtual Mouse
Bus 002 Device 001: ID 1d6b:0001 Linux Foundation 1.1 root hub

kb@ubuntu:~/Attify-Zigbee-Framework$ python main.py
[*] Executing Raven Refresh 
[*] pcapRefresh invoked 

device is not being located

**Comment by adi0x90** [comment]
Can you try `sudo ./zbid` with the `newkillerbee/killerbee/tools/` located in the `tools` directory?

**Comment by 6d6176** [comment]
Yes, I’ve since been able to use killerbee I have not been able to get the attify killerbee framework to work neither in attifyos or a Ubuntu/Debian

But I would like to trouble you for a guide on flashing the killerbee firmware on to an atmel RZUsbstick. I am waiting for a bus pirate to arrive in the mail and would like to update to firmware version 4 posted recently.

**Comment by nalin-dubey** [comment]
I am going through the same trouble my RZUsbstick is not being detected in the attify framework.
[*] Executing Raven Refresh 
[*] pcapRefresh invoked 

Any help please.

**Comment by lemonking77** [comment]
I'm using Ti equipment, and I have the same problem. How can I solve it? Please help me，and the settings of the UI are useless.

Bus 001 Device 001: ID 1d6b:0002 Linux Foundation 2.0 root hub
Bus 002 Device 004: ID 0451:16ae Texas Instruments, Inc. 
Bus 002 Device 002: ID 80ee:0021 VirtualBox USB Tablet
Bus 002 Device 001: ID 1d6b:0001 Linux Foundation 1.1 root hub

[*] Executing Raven Refresh 
[*] pcapRefresh invoked

**Comment by 6d6176** [comment]
been a while sense, i've used kb, deven, you might need to flash the
killerbee framework on to your ti stick.
